examples/pdo_poller_example.py

```diff
@@ -0,0 +1,86 @@
+import time
+from typing import Dict, List, Union
+
+from ingeniamotion.enums import SensorType
+from ingeniamotion.motion_controller import MotionController
+
+
+def establish_coe_connection(mc: MotionController) -> None:
```

Comment:
Same as baudrate example, you can use our functions directly. Remember that exist these functions:

- connect_servo_ethercat_interface_index: https://distext.ingeniamc.com/doc/ingeniamotion/0.7.1/ingeniamotion/communication.html#ingeniamotion.communication.Communication.connect_servo_ethercat_interface_index
- connect_servo_ethercat_interface_ip: https://distext.ingeniamc.com/doc/ingeniamotion/0.7.1/ingeniamotion/communication.html#ingeniamotion.communication.Communication.connect_servo_ethercat_interface_ip